Name: Roam only
Version: 1.3.5 -1
Author: mamouton orginally created by wmv5
Description: Enable roam only in phone preference screen

--- .orig/usr/palm/applications/com.palm.app.phoneprefs/app/controllers/preflist-assistant.js
+++ /usr/palm/applications/com.palm.app.phoneprefs/app/controllers/preflist-assistant.js
@@ -233,7 +233,8 @@
 		
 		this.voiceRoamingChoices =  [
 			{label: this.carrierOnlyLabel,value: "homeonly"},
-			{label : $L("Automatic"),value: "any"}
+			{label : $L("Automatic"),value: "any"},
+			{label : $L("Roam Only"),value: "roamonly"}
 		],
 	
 		this.voiceRoamingModel = {
@@ -1465,13 +1466,13 @@
 			// if update happens while we are open and band selection
 			// was previously allowed (assuming based on choices == 3)
 			// put things back
-			if (this.voiceRoamingChoices.length == 3) {
-				this.voiceRoamingChoices[1] = {
-					label: $L("Automatic"),
-					value: "any"	
-				};
-				this.voiceRoamingChoices.pop();
-			}
+			//if (this.voiceRoamingChoices.length == 3) {
+				//this.voiceRoamingChoices[1] = {
+					//label: $L("Automatic"),
+					//value: "any"	
+				//};
+				//this.voiceRoamingChoices.pop();
+			//}
 			
 			if (payload.extended.mode == 'any') {
 				this.voiceRoamingModel.currentVoiceRoaming = "any";
@@ -1477,8 +1478,8 @@
 				this.voiceRoamingModel.currentVoiceRoaming = "any";
 				$('dataroamingrow').show();
 			} else {
-				this.voiceRoamingModel.currentVoiceRoaming = "homeonly";
-				$('dataroamingrow').hide();
+				this.voiceRoamingModel.currentVoiceRoaming = "roamonly";
+				$('dataroamingrow').show();
 			}
 		}        
         this.controller.modelChanged(this.voiceRoamingModel);		  
